app/Infrastructure/Console/Kernel.php

```diff
@@ -36,6 +38,10 @@ protected function schedule(Schedule $schedule): void
             ->weekdays()
             ->dailyAt("08:30");
 
+        $schedule->command(SendNotificationAboutUpcomingAndOverdueMedicalExams::class)->monthlyOn(1, "08:00");
```

Comment:
```suggestion
        $schedule->command(SendNotificationAboutUpcomingAndOverdueMedicalExams::class)->monthlyOn(1, "08:00")->onOneServer();
```

Please change it in other command too: https://laravel.com/docs/10.x/scheduling#running-tasks-on-one-server